modelon/impact/client/entities/experiment.py

```diff
@@ -374,6 +430,37 @@ def get_trajectories(self, variables: List[str]) -> Dict[str, Any]:
             for j in case_nbrs
         }
 
+    def get_last_time_point(self, variables: List[str]) -> ExperimentResultPoint:
```

Comment:
I checked with Peter: as this should be valid for "steady state" and other CF as well, let's call the method `get_last_point`.

Is it possible to have default for "variables" as `None` to get all variables?

Reply:
Yes it is. Done. thanks!